src/jit/gentree.cpp

```diff
@@ -12002,16 +12002,22 @@ void Compiler::gtDispTreeRange(LIR::Range& containingRange, GenTree* tree)
 //
 // Arguments:
 //    node - the LIR node to dump.
+//    prefixMsg - an optional prefix for each line of output.
 //
-void Compiler::gtDispLIRNode(GenTree* node)
+void Compiler::gtDispLIRNode(GenTree* node, const char* prefixMsg /* = nullptr */)
 {
-    auto displayOperand = [](GenTree* operand, const char* message, IndentInfo operandArc, IndentStack& indentStack) {
+    auto displayOperand = [](GenTree* operand, const char* message, IndentInfo operandArc, IndentStack& indentStack, const char* prefixMsg = nullptr) {
         assert(operand != nullptr);
         assert(message != nullptr);
 
+        if (prefixMsg != nullptr)
+        {
+            printf("%s", prefixMsg);
```

Comment:
Minor nit: you could capture the length of `prefixMsg` and use it to adjust the alignment below. I'd be interested to see how the codegen dumps looked in both cases.